tests/shell/BashateBearTest.py

```diff
@@ -0,0 +1,40 @@
+from bears.shell.BashateBear import BashateBear
+from coalib.testing.LocalBearTestHelper import verify_local_bear
+
+# E002: indents can only be spaces
+E002_bad = """
+  echo "E002: Has a tab"
+"""
+
+# E003: indents should be multiples of 4
+E003_bad = """
+   indent is not multiple of 4
+"""
+
+# E006: too long lines
+E006_bad = 'a = "{}"\n'.format('a' * 100)
+
```

Comment:
From the comment before: Would put another blank line here

Reply:
where exactly do you want me to add a newline?

Reply:
exactly here :D